extension-basics.md

```diff
@@ -3,6 +3,34 @@ layout: sidebar
 title: Extension Basics
 permalink: /extension-basics/
 published: false
+tags: label distribution install remove
+contributors: caitmuenster muffinresearch bqbn
+author: caitmuenster
+date: 2019-06-24 09:00:00
 ---
 
+<!-- Overview Page Hero Banner -->
+
+<section class="overview-hero" style="background-image: url({% asset "develop-overview-hero-bg.jpg" @optim @path %});">
```

Comment:
Inline style attributes won't be allowed by CSP.

Reply:
Phew. I think it's a perfectly legitimate use of inline styles in this case, since we want a background image that can be assigned by a page creator without needing to edit the stylesheets.